examples/tutorials/hello-gltf/app.ts

```diff
@@ -0,0 +1,88 @@
+import {AnimationLoopTemplate, AnimationProps, Model} from '@luma.gl/engine/index';
+import {Device, load} from '@loaders.gl/core';
+import {GLTFLoader, postProcessGLTF} from '@loaders.gl/gltf';
+import {createGLTFObjects} from '@luma.gl/gltf/index';
+import {Matrix4} from '@math.gl/core';
+
+const INFO_HTML = `
+Have to start somewhere...
+`;
+
+const lightSources = {
+  ambientLight: { color: [255, 133, 133], intensity: 1 },
+  directionalLights: [ { color: [222, 244, 255], direction: [1, -0.5, 0.5], intensity: 10 } ],
+  pointLights: [ { color: [255, 222, 222], position: [3, 10, 0], attenuation: [0, 0, 0.01], intensity: 5 } ]
+}
+
+export default class AppAnimationLoopTemplate extends AnimationLoopTemplate {
+  static info = INFO_HTML;
+
+  device: Device;
+  scenes = [];
+  center = [0, 0, 0];
+  vantage = [0, 0, 0];
+  time: number = 0;
+
+  constructor({device}: AnimationProps) {
+    super();
+    this.device = device;
+    this.loadGLTF('Avocado');
+    const modelSelector = document.getElementById('model-select')
+    
+    modelSelector.addEventListener('change', e => {
+      this.loadGLTF((e.target as HTMLSelectElement).value);
+    });
+  }
+
+  onFinalize() {
+    this.scenes[0].traverse(({model}) => model.destroy());
+  }
+
+  onRender({aspect, device, time}: AnimationProps): void {
+    if (!this.scenes?.length) return;
+    const renderPass = device.beginRenderPass({clearColor: [0, 0, 0, 1]});
+
+    const projectionMatrix = new Matrix4().perspective({fovy: Math.PI / 3, aspect, near: 0.01, far: 100});
+    const eye = [this.vantage[0] * Math.sin(0.001 * time), this.vantage[1], this.vantage[2] * Math.cos(0.001 * time)];
+    const viewMatrix = new Matrix4().lookAt({eye, center: this.center});
+
+    this.scenes[0].traverse((node, {worldMatrix}) => {
+      const {model} = node;
+      const u_MVPMatrix = new Matrix4(projectionMatrix).multiplyRight(viewMatrix).multiplyRight(worldMatrix);
+      model.setUniforms({
+        u_Camera: eye,
+        u_MVPMatrix,
+        u_ModelMatrix: worldMatrix,
+        u_NormalMatrix: new Matrix4(worldMatrix).invert().transpose()
+      })
+
+      model.updateModuleSettings({lightSources});
+      model.draw(renderPass);
+    });
+    renderPass.end();
+  }
+
+  async loadGLTF(modelName: string) {
+    const {canvas} = this.device.canvasContext;
+    canvas.style.opacity = 0.1;
+
+    const gltf = await load(`https://github.khronos.org/glTF-Sample-Viewer-Release/assets/models/Models/${modelName}/glTF/${modelName}.gltf`, GLTFLoader);
+    const processedGLTF = postProcessGLTF(gltf);
+
+    const options = { pbrDebug: false, imageBasedLightingEnvironment: null, lights: true };
+    const {scenes} = createGLTFObjects(this.device, processedGLTF, options);
+    this.scenes = scenes;
+
+    // Calculate nice camera view
```

Comment:
I wonder if this is generic enough thatshould be done by the gltf module as a courtesy? I.e. calculate a `boundingBox` property?